src/test/resources/mappings/users.json

```diff
@@ -52,6 +52,19 @@
         }
       }
     },
+    {
+      "request": {
+        "method": "GET",
+        "url": "/users?query=id==21932a85-bd00-446b-9565-46e0c1a5490b"
```

Comment:
x-okapi-token/x-okapi-tenant will be useful here just to be sure that correct values were populated during FeignClient processing.